src/codeu/chat/client/core/View.java

```diff
@@ -42,11 +42,17 @@
   private final static Logger.Log LOG = Logger.newLog(View.class);
 
   private final ConnectionSource source;
+  private User user;
 
   public View(ConnectionSource source) {
     this.source = source;
   }
```

Comment:
A null user means an "unauthenticated" User. Hopefully the client does not try to perform a function that requires sending in the user id.